monitoring/pra/alerts.yaml

```
@@ -0,0 +1,79 @@
+x-inputs:
+  - name: namespace
+    type: constant
+    value: zenko
+  - name: kafka_connect_src_job
+    type: config
+    value: artesca-data-dr-source-base-queue-connector-metrics
+  - name: kafka_connect_sink_job
+    type: config
+    value: artesca-data-dr-base-queue-connector-metrics
+  - name: dr_sink_instance
+    type: config
+    value: artesca-data-dr
+  - name: RTO
+    type: constant
+    value:  86400 # 24 hours
+  - name: mongo_jobs
+    type: config
+    value: zenko/data-db-mongodb-sharded-shard.*
+
+groups:
+- name: PraAlerts
+  rules:
+
+  - alert: DrResourcePausedWhileOtherRunning
+    expr: |
+      (sum(up{job="${kafka_connect_src_job}", namespace="${namespace}"}) OR vector(0)) != (sum(up{job="${kafka_connect_sink_job}", namespace="${namespace}"}) OR vector(0))
```

~~nit: outside parenthesis not needed~~

not readable won a single line, please wrap & indent!

```suggestion
      (sum(up{job="${kafka_connect_src_job}", namespace="${namespace}"}) or vector(0))
      !=
      (sum(up{job="${kafka_connect_sink_job}", namespace="${namespace}"}) or vector(0))
```